iocore/net/quic/test/test_QUICStreamManager.cc

```diff
@@ -165,3 +165,85 @@ TEST_CASE("QUICStreamManager_total_offset_sent", "[quic]")
   sleep(2);
   CHECK(sm.total_offset_sent() == 1);
 }
+
+TEST_CASE("QUICStreamManager_stream_last_offset", "[quic]")
+{
+  MockQUICFrameTransmitter tx;
+  QUICApplicationMap app_map;
+  QUICError err;
+  MockQUICApplication mock_app;
+  app_map.set_default(&mock_app);
+  QUICStreamManager sm(&tx, &app_map);
+  std::shared_ptr<QUICTransportParameters> local_tp = std::make_shared<QUICTransportParametersInEncryptedExtensions>();
+  local_tp->add(QUICTransportParameterId::INITIAL_MAX_STREAM_DATA,
+                std::unique_ptr<QUICTransportParameterValue>(new QUICTransportParameterValue(4096, 4)));
+  std::shared_ptr<QUICTransportParameters> remote_tp =
+    std::make_shared<QUICTransportParametersInClientHello>(static_cast<QUICVersion>(0), static_cast<QUICVersion>(0));
+
+  sm.init_flow_control_params(local_tp, remote_tp);
+  char data[1024] = {0};
```

Comment:
Let's make this `uint8_t data[1024]`, then we can remove the `reinterpret_cast`s.